types/tryghost__content-api/index.d.ts

```diff
@@ -59,6 +59,7 @@ export interface Settings extends Metadata, CodeInjection, SocialMedia {
     description?: string | undefined;
     logo?: string | undefined;
     icon?: string | undefined;
+    accent_color?: Nullable<string> | undefined;
```

Comment:
I am not sure but i think you want this to be `string | undefined` or pls explain why it is different from other properties above.

Reply:
Yes, you are right. Firstly, I think `string | undefined` is the right type. But when I [check content-api API docs](https://ghost.org/docs/content-api/#settings) 

The settings also return null.
```bash
GET /content/settings/

```
The `accent_color` colour return null if the value does not exist. Then I check the type definition code. They handle null value with `Nullable<string> | undefined;`. So I used it. With the `Nullable<string> | undefined;` code, I handle null, undefined and string types. I think it is the best way. 

```bash
{
    "settings": {
        "title": "Ghost",
        "description": "The professional publishing platform",
        "logo": "https://docs.ghost.io/content/images/2014/09/Ghost-Transparent-for-DARK-BG.png",
        "icon": "https://docs.ghost.io/content/images/2017/07/favicon.png",
        "accent_color": null,
        "cover_image": "https://docs.ghost.io/content/images/2019/10/publication-cover.png",
        "facebook": "ghost",
        "twitter": "@tryghost",
        "lang": "en",
        "timezone": "Etc/UTC",
        "codeinjection_head": null,
        "codeinjection_foot": "<script src=\"//rum-static.pingdom.net/pa-5d8850cd3a70310008000482.js\" async></script>",
        "navigation": [
            {
                "label": "Home",
                "url": "/"
            },
            {
                "label": "About",
                "url": "/about/"
            },
            {
                "label": "Getting Started",
                "url": "/tag/getting-started/"
            },
            {
                "label": "Try Ghost",
                "url": "https://ghost.org"
            }
        ],
        "secondary_navigation": [],
        "meta_title": null,
        "meta_description": null,
        "og_image": null,
        "og_title": null,
        "og_description": null,
        "twitter_image": null,
        "twitter_title": null,
        "twitter_description": null,
        "members_support_address": "noreply@docs.ghost.io",
        "url": "https://docs.ghost.io/"
    }
}
```

Reply:
If you think `Nullable<string> | undefined;` is not the right approach. then I replace it with the `string | undefined` type.

Reply:
Will let  @knguyen0125 @antonve @maveric1977 @thrymgjol determine how to proceed with this.

Reply:
Hi @sheetalkamat, you are Right `string | undefined;` is the correct type. The docs mention in `https://ghost.org/docs/content-api/#settings` is outdated. I fixed the type and resubmitted the pull request.